### PR TITLE
Reflecting Changes to Service in config 95

### PR DESCRIPTION
**Pull request by ChristopherTulip**
With config 95 IT admins no longer need to request that Project and Visio are added to the service since it will be done automatically. I have adjusted the docs to reflect that and adjusted the workflow accordingly.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/managed-desktop/get-started/project-visio.md [hunk 1]
```diff
@@ -23,26 +23,17 @@ Admins should verify that they meet these prerequisites:
 - **Company Portal** -  The Company Portal must be available in your tenant for your users to install these applications. If the Company Portal isn’t deployed in your tenant, see [Company Portal](company-portal.md).
 
 ## Deploy Project and Visio for Microsoft Managed Desktop devices
-After you submit your support request, Microsoft Managed Desktop will create three Azure AD groups and three application deployments through Microsoft Intune to deploy the apps to your tenant.  
+Microsoft Managed Desktop will add Microsoft Project and Microsoft Visio as two Win32 Applications in Microsoft Intune. We will also create two groups in Azure Active Directory which will be assigned to the coresponding application with the "Available" intent.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/get-started/project-visio.md:26]:
```suggestion
Microsoft Managed Desktop will add Microsoft Project and Microsoft Visio as two Win32 Applications in Microsoft Intune. We will also create two groups in Azure Active Directory which will be assigned to the corresponding application with the **Available** assignment type. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by mypil** [comment]
@ChristopherTulip - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

File: microsoft-365/managed-desktop/get-started/project-visio.md (hunk 1 above)
Comment (ChristopherTulip) [reply microsoft-365/managed-desktop/get-started/project-visio.md:26]: Thanks for this good catches, the other edits we should make available bold and change "intent" to "assignment type" per the intune language: 

We will also create two groups in Azure Active Directory which will be assigned to the corresponding application with the **Available** assignment type.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@ChristopherTulip 's edits included.

**Pull request merged**